css/index.css

```
@@ -1,3 +1,458 @@
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+.container {
+	max-width: 880px;
+	/* TB LR */
+	margin: 0 auto;
+	/* border: 1px solid red; */
+	display: flex;
+	flex-direction: column;
+  }
+
+  @media ( max-width: 800px ) {
+      .container {
+          width: 100%;
+      }                
+  }
+  
+  header {
+      display: flex;
+    
+  }
+  @media ( max-width: 800px ) {
+    header {
+        flex-direction: column;
+        justify-content: space-around;
+    }
+  }
+  
+  header nav {
+	display: flex;
+	width: 691px;
+	justify-content: space-between;
+	margin-top: 35px;
+	padding-right: 100px;
+  }
+
+  @media ( max-width: 800px ) {
+      header nav {
+         width: 100%;
+         padding-right: 0;
+         justify-content: space-around;
+      }
+  }
+
+  @media ( max-width: 500px ) {
+      header nav {
+          flex-direction: column;
+          align-items: center;
+      }
+  }
+  
+  header nav a {
+	display: flex;
+	margin-right: 65px;
+	font-size: 14px;
+	color: gray;
+	text-decoration: none;
+  }
+  
+  @media ( max-width: 500px ) {
+      header nav a {
+        padding: 5%;
+        font-size: 1.75rem;
+        margin-right: 0;
+        border-bottom: 1px solid gray;
+        width: 100%;
+        justify-content: center;
+      }
+  }
+  header nav a:hover {
+	text-decoration: underline;
+  }
+  
+  header nav a:first-child {
+	margin-left: 0;
+  }
+
+  @media (max-width: 500px) {
+      header nav a:first-child {
+          border-top: 1px solid gray;
+      }
+  }
+  
+  header nav a:last-child {
+	margin-right: 0;
+  }
+  
+  header .logo {
+	display: flex;
+	margin-top: 25px;
+	height: auto;
+  }
+  
+  @media ( max-width: 800px ) {
+    .logo {
+        display: flex;
+        align-self: center;
+        order: -1;
+        max-width: 25%;
+    }
+  }
+
+  @media ( max-width: 500px ) {
+      .logo {
+          max-width: 50%;
+          margin: auto;
+      }
+  }
+  /* cta */
+
+  
+  .cta {
+	  display: flex;
+	  justify-content: space-evenly;
+  }
+  
+  .cta-text {
+	/* border: 1px solid red; */
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	vertical-align: top;
+	font-size: 77px;
+	margin-top: 120px;
+	text-align: center;
+	padding: 0px 75px;
+  }
```

```suggestion
	@media (max-width: 500px) {
		.cta-text {
			width: 100%
		}
  	}
```
This will make the page have no horizontal scroll bar